test/unittests/evm_other_test.cpp

```diff
@@ -56,3 +56,12 @@ TEST_F(evm_other, loop_full_of_jumpdests)
     execute(code);
     EXPECT_GAS_USED(EVMC_SUCCESS, 7987882);
 }
+
+TEST_F(evm_other, solidity_deploy)
```

Comment:
If you are looking for a more descriptive name for this unit test, you could perhaps use the name `abiv2_3d_bytes1_array_deploy`. The underlying contract is checks if abiv2 coding works as expected for a 3 dimensional `bytes1` array of type `bytes1[][4][4]`. The contract does not have an explicit constructor.